ui/lib/core/addon/components/info-table-row.hbs

```diff
@@ -90,18 +87,18 @@
                 <span class="is-word-break has-text-black" data-test-row-value={{@label}}>{{@value}}</span>
               </T.Trigger>
               <T.Content @defaultClass="tool-tip">
-                <CopyButton
-                  @clipboardText={{@tooltipText}}
-                  @success={{action (set-flash-message "Data copied!")}}
-                  @tagName="div"
-                  disabled={{not @isTooltipCopyable}}
-                  class={{if @isTooltipCopyable "has-pointer"}}
-                  data-test-tooltip-copy
-                >
-                  <div class="box">
-                    {{@tooltipText}}
-                  </div>
-                </CopyButton>
+                <div class="box is-flex-center">
```

Comment:
<img width="178" alt="Screenshot 2023-09-14 at 3 03 55 PM" src="https://github.com/hashicorp/vault/assets/68122737/32aa293b-6d39-4186-acad-01b4e06f0fd0">

Reply:
Wow, copy in a tooltip seems like a serious anti-pattern but I guess that's what we had before 🙈

Reply:
haha, my thoughts exactly! We created a polishing ticket to consider redesigning this when we initially came across it